packages/rich-text/src/component/index.native.js

```diff
@@ -860,15 +873,31 @@ export class RichText extends Component {
 					selectionColor={ this.props.selectionColor }
 				/>
 				{ isSelected && (
-					<FormatEdit
-						formatTypes={ formatTypes }
-						value={ record }
-						withoutInteractiveFormatting={
-							withoutInteractiveFormatting
-						}
-						onChange={ this.onFormatChange }
-						onFocus={ () => {} }
-					/>
+					<>
+						<FormatEdit
+							formatTypes={ formatTypes }
+							value={ record }
+							withoutInteractiveFormatting={
+								withoutInteractiveFormatting
+							}
+							onChange={ this.onFormatChange }
+							onFocus={ () => {} }
+						/>
+						<BlockFormatControls>
+							<ToolbarButton
+								title={ __( 'Insert mention' ) }
+								icon={ <Icon icon={ atSymbol } /> }
+								onClick={ () => {
+									addMention().then( ( mentionUserId ) => {
+										this.insertString(
+											record,
+											`@${ mentionUserId }`
+										);
+									} );
```

Comment:
I'm not sure if it's very important but I'm getting a warning on iOS the first time I cancel the mention selection in the console saying `Possible Unhandled Promise Rejection` I think because it's not handled here :)

Reply:
Good catch I now added an empty handler to remove this warning.